src/status_im2/contexts/syncing/events.cljs

```diff
@@ -74,12 +75,19 @@
 
 (rf/defn preparations-for-connection-string
   {:events [:syncing/get-connection-string-for-bootstrapping-another-device]}
-  [{:keys [db]} entered-password set-code]
+  [{:keys [db]} entered-password on-valid-connection-string]
   (let [valid-password? (>= (count entered-password) constants/min-password-length)
-        show-sheet      (fn [connection-string]
-                          (set-code connection-string)
-                          (rf/dispatch [:syncing/update-role constants/local-pairing-role-sender])
-                          (rf/dispatch [:bottom-sheet/hide]))]
+        show-sheet      (fn [response]
+                          (if (sync-utils/valid-connection-string? response)
+                            (do
+                              (on-valid-connection-string response)
+                              (rf/dispatch [:syncing/update-role constants/local-pairing-role-sender])
+                              (rf/dispatch [:hide-bottom-sheet]))
+                            {:db (update db
```

Comment:
@ilmotta @cammellos I am trying to get the error message show on the input when the user puts in a wrong password.

in the case of syncing setup this seems to be the right place but this approach isn't working. 
Any thoughts how I can/should do this? 🤔

Reply:
this is all handled now 👍